gym_roboy/envs/msj_env.py

```diff
@@ -79,4 +83,4 @@ def _did_reach_goal(self, actual_joint_angle) -> bool:
 
     @property
     def _l2_distance_for_success(self):
-        return _l2_distance(self.action_space.low, self.action_space.high) / 100  # 100 seems reasonable
+        return _l2_distance(self.observation_space.low, self.observation_space.high) / 1000  # 100 seems reasonable
```

Comment:
Seems like the comment did not get updated.
This is why I (and many Software Engineers) think that comments are bad :)

Reply:
I updated the number just based on experiments. Decided to stick with 1000. I assumed your comment might have based on calculation. Thus might be needed in the future.

Reply:
haha the `100` was just any number. Please remove it :)

Reply:
Also, there is a small error in the l2 distance for sucess. It is taking into account the whole observation space, which right now is bounded by pi. But in a change in #16, I actually set the `joint_vel` as unbounded (since thats more accurate) and therefore this l2 distance would become infinite. Instead of being based on the entire observation space, it should be based on the `joint_angle` bounds only.

Reply:
^^ true